BrainPortal/lib/session_helpers.rb

```diff
@@ -52,6 +52,15 @@ def current_project
     @current_project
   end
 
+  # return currently active project id if user can assign to it or own group id. Serves as default destination project for many operation
```

Comment:
Use a nice uppercase letter to start your comment block? Split it to make it shorter a little?

Reply:
corrected